src/js/index.js

```diff
@@ -7,42 +7,20 @@
 import ready from 'lite-ready';
 import { checkForUser } from './userAuth';
 
-
-/**
- * Converts an input to a button in order to improve accessibility.
- *
- * @param {string} selector
- */
-const convertInputToButton = selector => {
-    const input = document.querySelector(selector);
-
-    if (input) {
-        const replaceTag = input.outerHTML.replace(/^<input/, '<button');
-        input.outerHTML = `${replaceTag}</button>`;
-
-        // Query the DOM again for this element now it has changed to a button
-        const button = document.querySelector(selector);
-        button.setAttribute('type', 'button');
-
-        return button;
-    }
-
-    return null;
-};
-
 /**
  * Setup the behaviour for the header component.
  */
 const setupHeader = () => {
-    const menuButton = convertInputToButton('[data-nav-enhance]');
+    const menuButton = document.querySelector('[data-nav-button]');
+    const accessibleButton = document.querySelector('[data-nav-accessible-button]');
+
+    if (accessibleButton) {
+        accessibleButton.classList.remove('is-shown--noJS');
+        accessibleButton.classList.add('is-hidden');
```

Comment:
Think you could get away with not needing this block if you added a class `c-nav-trigger--noJS` to the input and styled it as `display: none; visibility: hidden;`.

That way, it should hide the element by default and override it if `is-shown--noJS` styles kick in.

Reply:
I didnt need any of that - I forgot the use of the `--noJS` classes 😄